src/category_theory/natural_isomorphism.lean

```diff
@@ -92,23 +93,42 @@ by tidy
 
 end category_theory.nat_iso
 
-namespace category_theory.functor
+open category_theory
 
-universes u₁ u₂ v₁ v₂
+namespace category_theory.functor
 
 section
 variables {C : Sort u₁} [𝒞 : category.{v₁} C]
           {D : Sort u₂} [𝒟 : category.{v₂} D]
 include 𝒞 𝒟
 
+section
+variables {E : Sort u₃} [ℰ : category.{v₃} E]
+include ℰ
+
+def map_nat_iso {F G : C ⥤ D} (K : D ⥤ E) (α : F ≅ G) : (F ⋙ K) ≅ (G ⋙ K) :=
```

Comment:
I would have expected this to have "whisker" in the name, and maybe live alongside the other whiskering operations and defined in a similar way.

I think I have similar definitions in one of my projects, and later tonight I'll try to take a shot at figuring out the best setup here.

Reply:
Ok, I agree it's dumb naming. I've removed this for now.